api/libs/oauth_data_source.py

```diff
@@ -154,11 +154,11 @@ def get_authorized_pages(self, access_token: str):
         for page_result in page_results:
             page_id = page_result['id']
             page_name = 'Untitled'
-            for key in ['Name', 'title', 'Title', 'Page']:
```

Comment:
When the language settings in Notion were changed, the values for [‘Name’, ‘title’, ‘Title’, ‘Page’] differed depending on the language, causing page_name not to be retrieved correctly.